MREUnityRuntime/MREUnityRuntimeLib/Core/Collider.cs

```diff
@@ -118,13 +149,27 @@ internal ColliderPatch GenerateInitialPatch()
                     Center = boxCollider.center.CreateMWVector3()
                 };
             }
-            else if (_collider is MeshCollider meshCollider)
-            {
-                colliderGeo = new MeshColliderGeometry();
-            }
             else if (_collider is CapsuleCollider capsuleCollider)
             {
-                colliderGeo = new CapsuleColliderGeometry();
+                MWVector3 size;
+                if (capsuleCollider.direction == 0)
+                {
+                    size = new MWVector3(capsuleCollider.height, 2 * capsuleCollider.radius, 2 * capsuleCollider.radius);
```

Comment:
>2 *  [](start = 65, length = 4)

Curious what the need is here for all the doubling happening to the values in Unity before we send them up in the initial patch to the app?

Reply:
I've standardized the capsule dimensions behind a bounding box, so radius is size / 2

---
In reply to: [316738178](https://github.com/microsoft/mixed-reality-extension-unity/pull/98#discussion_r316738178) [](ancestors = 316738178)

Reply:
Please add a comment here documenting this for when someone else is in here in the future.  Thanks